infra/aws/index.ts

```diff
@@ -955,6 +955,8 @@ export const dailyDataDumpTaskDefinition = createScheduledTask(
       "python",
       "manage.py",
       "scorer_dump_data",
+      "--batch-size 1000",
+      "--database read_replica_0",
```

Comment:
I am not 100% sure this will work ... can you specify with '=' in between?
  `--database=read_replica_0`,

Reply:
FYI : Works both ways: `--database=read_replica_0` & `--database read_replica_0`(tested locally)